backend/app/models/primary_donor.rb

```diff
@@ -1,10 +1,9 @@
 # frozen_string_literal: true
 
 class PrimaryDonor < ApplicationRecord
-  has_one_base64_attached :image
-
   has_many :campaigns, dependent: :destroy
 
   validates :name, presence: true
   validates :email, presence: true, uniqueness: true, format: { with: URI::MailTo::EMAIL_REGEXP }
+  # validates :image_url, presence: true, allow_blank: false
```

Comment:
These validations + database constraints should be added back once we have migrated the data over.

Reply:
How will we be migrating the data?